hafnian/samples.py

```diff
@@ -458,3 +466,53 @@ def torontonian_sample_graph(A, n_mean, samples=1, max_photons=30, pool=False):
     Q = gen_Qmat_from_graph(A, n_mean)
     cov = Covmat(Q)
     return torontonian_sample_state(cov, samples, hbar=2, max_photons=max_photons, pool=pool)
+
+
+def hafnian_sample_classical_state(cov, samples, mean=None, hbar=2, sigdigits=6):
+    r""" Returns samples from a Gaussian state that has a positive P function
+    Args:
+        cov(array): a :math:`2N\times 2N` ``np.float64`` covariance matrix
+            representing an :math:`N` mode quantum state. This can be obtained
+            via the ``scovmavxp`` method of the Gaussian backend of Strawberry Fields.
```

Comment:
Should we delete the part that says 'this can be obtained...'?

Reply:
I actually like to keep there; in fact the reason why I added that line is precisely because I always forget if I should use scovmat or scovmatxp...